src/components/alert/alert.component.tsx

```diff
@@ -5,9 +5,16 @@ import Dialog, { DialogProps } from "../dialog";
 export const Alert = ({
   children,
   size = "extra-small",
+  topModalOverride,
   ...rest
 }: DialogProps) => (
-  <Dialog data-component="alert" role="alertdialog" size={size} {...rest}>
+  <Dialog
+    data-component="alert"
+    role="alertdialog"
+    size={size}
+    topModalOverride={topModalOverride}
```

Comment:
Yeah it can be passed via `...rest` but I'm leaning towards preferring we explicitly destructure the props as it's more readable to people working on carbon without much experience of it